clients/spark/project/plugins.sbt

```diff
@@ -1,2 +1,4 @@
 addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.15.0")
 addSbtPlugin("com.github.sbt" % "sbt-pgp" % "2.1.2")
+addSbtPlugin("laughedelic" % "sbt-publish-more" % "0.1.0")
```

Comment:
Do we still need this?

Reply:
Yes - this is what allows us to publish to multiple repos (sonatype + s3)